packages/keyboard-direction-mixin/test/keyboard-direction-mixin.test.ts

```diff
@@ -25,11 +25,41 @@ class KdmElement extends KeyboardDirectionMixin(SlottedItemsMixin(LitElement)) {
       </div>
     `;
   }
+
+  _isVertical() {
+    return true;
+  }
+}
+
+@customElement('kdm-element-horizontal')
+class KdmElementHorizontal extends KeyboardDirectionMixin(SlottedItemsMixin(LitElement)) {
+  render() {
+    return html`
+      <style>
+        :host {
+          display: block;
+        }
+
+        div {
+          display: flex;
+        }
+      </style>
+      <div>
+        <slot></slot>
+      </div>
+    `;
+  }
+
+  _isVertical() {
+    return false;
+  }
 }
 
 describe('KeyboardDirectionMixin', () => {
-  let element: KdmElement;
+  let element: KdmElementVertical;
```

Comment:
Let's create two sub-suites:

- `vertical mode` 
- `horizontal mode`

Inside of those there should be `default` and `RTL` sub-suites.

Reply:
Done